### ISSUE TITLE
[Ally] [VoiceOver] Unable to select day in datepicker control.

### ISSUE THREAD

**Issue by nadokupa745** [OfficeDev/office-ui-fabric-react#11273]
### Environment Information

- **Package version(s)**: ( office-ui-fabric-react - 7.63.0)
- **Browser and OS versions**: (Safari and latest IOS )

### Describe the issue:
When voice over is enabled in Iphone, user is unable to select a day since focus is coming to select only on month but not the day.

#### Actual behavior:
Focus is not coming to day.

#### Expected behavior:
Focus should be on the day so that user can select day.

**Comment by aneeshack4** [comment]
Hi @nadokupa745, thanks for reporting this! I see it does repro - it's able to put focus on the day headings too like "W":
![image](https://user-images.githubusercontent.com/4161791/69469435-b2175700-0d45-11ea-8d6a-830387808d52.png)
but not on the numbers of the days:
![image](https://user-images.githubusercontent.com/4161791/69469441-c8251780-0d45-11ea-985a-4839e4aaab42.png)
It doesn't repro on Mac + chrome but it does on iOS + Chrome (and Safari which is the above screenshots).
iOS + chrome:
![image](https://user-images.githubusercontent.com/4161791/69469628-bb54f380-0d46-11ea-81c4-3a63b767a85b.png)
Mac + Chrome (no repro):
![image](https://user-images.githubusercontent.com/4161791/69469633-c4de5b80-0d46-11ea-9bef-68e73b7d699d.png)
So this seems like an iOS issue... @joschect what do you think?

**Comment by nadokupa745** [comment]
@aneeshack4 , can you pls let me know if this issue has been prioritized?

**Comment by aneeshack4** [comment]
@nadokupa745 I've already shown that this doesn't repro on Mac but it does in iOS... I'll ping the DatePicker component owner offline to remind him to follow up.

**Comment by lorejoh12** [comment]
To be honest here, I'm not familiar with the iOS setup and don't have a device to repro this. If there's an iOS accessibility expert who would be able to help out here, I would appreciate that.

**Comment by aneeshack4** [comment]
This doesn't repro on Mac and only on iOS which is really weird. Going to reassign to @lorejoh12 on the calendar team so they can get to this when they have the cycles.

**Comment by lorejoh12** [comment]
I borrowed an iPhone and used codepen to build a minimum repro for the issue (since the Fabric deploy links don't really work on iPhones for some reason)

https://codepen.io/lorejoh12/pen/RwNaKyK

There are 4 buttons in the codepen. The top button is broken, the bottom three all work fine.

This actually looks like it might be an iOS VoiceOver bug- when you have a table structure with a button, and a span inside that button,
<td>
   <button>
      <span>text</span>
   </button>
</td>

having role=gridcell on the button AND aria-hidden=true on the span causes iOS VoiceOver to break. Having one or the other causes no issues.

It's not simple to remove this combination in the Fabric control. The role=gridcell helps Narrator announce the row/column information when using arrow keys, and aria-hidden prevents the text from being read twice. We should probably follow up with Apple here and report this.

**Comment by xugao** [comment]
Issue is unrelated to Fabric team but applies to another product. Closing
